.github/workflows/add_member.yml

```diff
@@ -68,9 +68,18 @@ jobs:
       - name: Create branch
         run: git checkout -b ${{ env.BRANCH_NAME }}
 
+      - name: Check if user wants to become a designer
+        id: check_designer
+        run: |          
+          if gh issue view ${{ env.ISSUE_NUMBER }} --json body | jq -e '.body | contains("Do you wish to join the designers team?")'; then
```

Comment:
Won't this appear for all of them? I think we'd need to look for the checked box. Something like `[x] Do you wish to join the designers team?`